tests/rules/test_cp_verify_alarm.py

```diff
@@ -0,0 +1,164 @@
+# This file is part of ts_watcher.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import types
+import unittest
+
+import yaml
+
+# from lsst.ts.idl.enums.Watcher import AlarmSeverity
+from lsst.ts import salobj
+from lsst.ts import watcher
+
+STD_TIMEOUT = 5  # Max time to send/receive a topic (seconds)
+
+
+class CpVerifyTestCase(unittest.IsolatedAsyncioTestCase):
+    def setUp(self):
+        salobj.set_random_lsst_dds_partition_prefix()
+
+    def make_config(self, name):
+        """Make a config for the Cp Verify rule.
+
+        Parameters
+        ----------
+        name : `str`
+            CSC name and index in the form `name` or `name:index`.
+            The default index is 0.
+        """
```

Comment:
I'm confused by this: `name` isn't a config parameter, and all three config parameters are missing. Shouldn't the arguments be:
* calibration_type
* ocps_index
* verification_threshold

Reply:
Ah, thanks! This probably was copied from some other alarm test many months ago. I'll change the arguments and the dictionary:  

        `config_dict = dict(calibration_type =  calibration_type,
                           ocps_index = ocps_index,
                           verification_threshold=verification_threshold
                      )`